docs/user-guide/images.md

```diff
@@ -100,6 +101,23 @@ Troubleshooting:
   - `plugins.go:56] Registering credential provider: aws-ecr-key`
   - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
+### Using Azure Container Registry (ACR)
+When using Azure container registry you can authenticate using either an admin user or a service principal.
+In either case, authentication is done via standard Docker authentication.  These instructions assume the
+python based [az](https://github.com/azure/azure-cli) tooling.
+`
+```console
+# To generate new credentials
+az acr credential renew -n ${CLUSTER_NAME} -g ${CLUSTER_RESOURCE_GROUP}
```

Comment:
This command will not work if ACR registry is configured to use a Service Principal (recommended). It assumes ACR Admin Account is used (which might not be available when it goes GA). IMO this documentation should just refer to ACR docs which documents: 1) how to create a registry 2) create a SP account 3) make RBAC assignment for SP to use ACR.